ipfs-cluster-ctl/formatters.go

```diff
@@ -82,23 +83,41 @@ func textFormatPrintIDSerial(obj *api.IDSerial) {
 	}
 
 	fmt.Printf("%s | %d peers\n", obj.ID, len(obj.ClusterPeers))
-	fmt.Println("  > Addresses:")
+	addrs := sort.StringSlice{}
```

Comment:
Ah, good point, forgot slices worked that way. Good fix too, I think it's better than what I suggested (because the `sort.Strings()` has an implicit cast that you're not doing now, not sure how costly that is but still).